### PR TITLE
HttpService and related classes emulated

### PR DESCRIPTION
**Pull request by nihavend**
@aharui may you merge pls.

### CODE REVIEW

**Comment by nihavend** [comment]
Hi @aharui, may you please merge my pull request.

File: frameworks/projects/MXRoyale/src/main/royale/MXRoyaleClasses.as [hunk 1]
```diff
@@ -68,6 +68,7 @@ internal class MXRoyaleClasses
 	import mx.effects.Tween; Tween;
 	import mx.system.ApplicationDomain; ApplicationDomain;
 	import mx.rpc.http.HTTPService; mx.rpc.http.HTTPService;
+	import mx.rpc.http.mxml.HTTPService; mx.rpc.http.HTTPService;
```
Comment (aharui) [inline frameworks/projects/MXRoyale/src/main/royale/MXRoyaleClasses.as]: This line shouldn't be necessary.  The mx-royale-manifest will take care of it.

**Comment by aharui** [comment]
Do we really need IRepeaterClient?  Also NameUtil.  Would be nice to not have to support them in the future.

**Comment by nihavend** [comment]
> Do we really need IRepeaterClient? Also NameUtil. Would be nice to not have to support them in the future.

I am not sure. Whan i was working on DefaultGridItemRenderer, i needed both and emulated. Than i gave for DefaultGridItemRenderer but these classes were ready and i included them in PR. I can remove if you say so.

File: frameworks/projects/MXRoyale/src/main/royale/MXRoyaleClasses.as (hunk 1 above)
Comment (nihavend) [reply frameworks/projects/MXRoyale/src/main/royale/MXRoyaleClasses.as]: Okay, i had missed adding mx-royale-manifest to compiler arguments.

Comment (nihavend) [reply frameworks/projects/MXRoyale/src/main/royale/MXRoyaleClasses.as]: Removed.

**Comment by aharui** [comment]
We should try not to include code we don't want to support forever.  I'm not sure NameUtil would be critical to anyone's application.  Repeater may not be easily implementable on other platforms/languages.  So let's remove them.

**Pull request merged**